### ISSUE TITLE
Example page with line chart tooltips - bug - wrong tooltip position

### ISSUE THREAD

**Issue by yairEO** [Globegitter/chartist-plugin-tooltip#23]
https://gionkunz.github.io/chartist-js/examples.html#behavior-with-jquery

The tooltips are not where they should be.

#### video GIF link:

http://i.giphy.com/xTiTngDRcOfB365AR2.gif

------------------
Tested on: *FF 40*, *Windows 7*
on Chrome tooltips are positioned well. I suggest you to use the **Tether** javascript lib for positioning.

**Comment by Globegitter** [comment]
@yairEO That should have been fixed in the latest version 0.0.11, see https://github.com/Globegitter/chartist-plugin-tooltip/pull/19 Maybe the example hasn't been updated?

**Comment by yairEO** [comment]
you're asking me? I thought you're in charge of it, since you wrote the code..

**Comment by Globegitter** [comment]
It was more a rhetorical question - I suppose that does not translate that well on the web :P

Quite busy right now but when I get a chance I'll check it out - thanks for reporting.

**Comment by vimmynegi** [comment]
Hello 
I have same problem with my chart and i analyze that this problem is occurring on window 8 system .. On windows 7 its working fine

**Comment by andrewdavidcostello** [comment]
I also had this issue, I fixed it with the following changes:

```
var left = event.pageX - box.left - window.pageXOffset ;
var top = event.pageY - box.top - window.pageYOffset ;
```

to

```
var left = event.pageX;
var top = event.pageY;
```

**Comment by ellenschwartau** [comment]
Will this fix be merged? I'm facing this issue with version 0.0.11, too.

**Comment by Globegitter** [comment]
@ellenschwartau the npm module under the name `chartist-plugins-tooltip` was published by someone else. Might try to contact the guy so I can publish under the old name.

**Comment by ealves-pt** [comment]
Seems to be fixed in the current release v0.0.13
